### PR TITLE
NR: Dialog/ Story Rewrite: Scenario One.

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
So I was talking in issue #7448 and someone said that I could make some dialog changes but thought I remembered some other editing problems in the story. I looked at the first scenario and a lot of things in the story that didn't make sense on reading through. I'd like to, if you don't mind, go through each scenario and do some editing.

I don't want my editing to hurt anyone's feelings, and I tried to separate each change in a different commit. So you can pick and choose the changes and squash the commits you like. If you don't like any of the changes, you can reject them all.

I edited it as well as I could, and I ran it through LanguageTool. A lot of the problems I found were that the sentences were too long and made my eyes cross. I tried to shorten them.

### CODE REVIEW

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 1]
```diff
@@ -20,7 +20,7 @@
             {PROLOGUE_STAGE1}
         [/part]
         [part]
-            story= _ "The dwarves of Knalga, cannily reckoning the gains from trade, built strong fortifications around Dwarven Doors. The people who called it home, dwarven and human alike, knew that the wealth of their city might arouse envy from afar — hence they kept their weapons sharp, and reckoned themselves well able to fight off any bandit gang or petty warlord that could arise in the thinly-settled Northlands."
+            story= _ "The dwarves of Knalga, cannily reckoning the gains from trade, built strong fortifications around Dwarven Doors. The people who called it home, dwarven and human alike, knew that the wealth of their city might arouse envy from afar — hence they kept their weapons shar and were well able to fight off any bandit, gang, or petty warlord that could arise in the thinly-settled Northlands."
```
Comment (cooljeanius) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:23]: now the "p" is missing from "sharp", though

**Review by cooljeanius — CHANGES_REQUESTED** [review]

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg (hunk 1 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:23]: Oops.

Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg]: Oops I fixed it. I'm not perfect either.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 2]
```diff
@@ -30,7 +30,7 @@
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "The raids were mere probes, at first, and the orcs mere rabble. But they grew more numerous, and threatening, and the raiding parties became war-bands and then companies. Then there arose a great warlord among the Bloody Sword tribe, the chieftain called Khazg Black-Tusk; and he raised an army, and besieged Dwarven Doors."
+            story= _ "At first, the orcs disorganized, a mere rabble that easily fell beneath their blade off. Little did the people of Dwarven Doors know that the attacks were merely a probing at their defenses. The raids they grew more numerous and threatening. And soon the raiding parties became war-bands and then companies. At last, one morning, the people of the dwarven doors arose to see a great warlord of the Bloody Sword tribe, Khazg Black-Tusk, and his army around their defenses."
```
Comment (cooljeanius) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg]: missing "were" before "disorganized"? Also, "fell beneath their blade off" sounds awkward... off of what? 3rd sentence: if changing "They" to "The raids", get rid of the original "they". 4th sentence: this is a bit too formal of a context to be starting a sentence with "And". 5th sentence: "Dwarven Doors" is a proper noun and requires capitalization.

Comment (cooljeanius) [reply data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg]: oh wait never mind I see now that this is outdated

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 3]
```diff
@@ -40,12 +40,12 @@
...
+            story= _ "After a year and a day of grim battle, the city fell. The streets of Dwarven Doors ran with gore. The dwarves managed to retreat into the caverns Knalga, but the humans who survived the sack were enslaved. Cut off from contact with the surface, the dwarves of Knalga fought on, but knew their subjugation could not be averted forever."
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "Dark years passed; human generations grew up in slavery, and came near to forgetting that their ancestors had been free — and might have completely forgotten were it not for a fateful day in 518YW. The day that changed the lives of the people of Dwarven Doors — forever."
+            story= _ "Dark years passed; human generations grew up and died in slavery, and with every passing year they and came close to forgetting that their ancestors had ever been free. They might have completely forgotten were it not for a fateful day. The day that changed the lives of the people of Dwarven Doors — forever."
```
Comment (cooljeanius) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:48]: Personally I'd put a comma after "passing year", but that's just my style. Also, "they and" sounds weird?

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 4]
```diff
@@ -15,49 +15,48 @@
 
     [story]
         [part]
-            story= _ "Far to the north of Wesnoth, and hard by the dwarven caves of Knalga, there was once a thriving mixed community known as Dwarven Doors. Food, wood, and cloth from all over Wesnoth and many elvish lands as well were brought here to be sold to the dwarves, while ores, metals and worked dwarven manufactures were bought here to be traded throughout the surface nations. Dwarves and humans worked and lived side by side; even the elves who came to trade mingled more with the other kindred races than was common elsewhere. Dwarven Doors grew populous and wealthy."
+            story= _ "Far to the north of Wesnoth, and close to dwarven caves of Knalga, there was once a thriving mixed community known as Dwarven Doors. Food, wood, and cloth from all over Wesnoth and many elvish lands as well were brought here to be sold to the dwarves, while ores, metals and worked dwarven manufactures were bought here to be traded throughout the surface nations. Dwarves and humans worked and lived side by side; even the elves who came to trade mingled more with the other kindred races than was common elsewhere. Dwarven Doors grew populous and wealthy."
```
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:18]: Sounds like the original was going for an olden style of English, which I understand Wesnoth as a whole goes for. I don't think this is a good change.

Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:18]: I agree with wedge. I don't think this is an improvement in this case.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 5]
```diff
@@ -15,49 +15,48 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1}
         [/part]
         [part]
-            story= _ "The dwarves of Knalga, cannily reckoning the gains from trade, built strong fortifications around Dwarven Doors. The people who called it home, dwarven and human alike, knew that the wealth of their city might arouse envy from afar — hence they kept their weapons sharp, and reckoned themselves well able to fight off any bandit gang or petty warlord that could arise in the thinly-settled Northlands."
+            story= _ "The dwarves of Knalga, cannily reckoning the gains from trade, built strong fortifications around Dwarven Doors. The people who called it home, dwarven and human alike, knew that the wealth of their city might arouse envy from afar — hence they kept their weapons sharp and were well able to fight off any bandit, gang, or petty warlord that could arise in the thinly-settled Northlands."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:23]: Ditto here.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 6]
```diff
@@ -15,49 +15,48 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "For many years it was indeed so. Dwarven Doors was a peaceful and prosperous place. Until the orcs came..."
+            story= _ "It went on for many years and Dwarven Doors was a peaceful and prosperous place, that is, until the orcs came..."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:28]: And here.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 7]
```diff
@@ -15,49 +15,48 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "The raids were mere probes, at first, and the orcs mere rabble. But they grew more numerous, and threatening, and the raiding parties became war-bands and then companies. Then there arose a great warlord among the Bloody Sword tribe, the chieftain called Khazg Black-Tusk; and he raised an army, and besieged Dwarven Doors."
+            story= _ "At first, the orcs disorganized, a mere rabble that easily fell beneath their blade off. Little did the people of Dwarven Doors know that the attacks were merely a probing at their defenses. The raids they grew more numerous and threatening. And soon the raiding parties became war-bands and then companies. At last, one morning, the people of the Dwarven Doors arose to see a great warlord of the Bloody Sword tribe, Khazg Black-Tusk, and his army around their defenses."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:33]: I'm having an extremely hard time reading your edit. Maybe use simpler wording or sentence structure here?

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 8]
```diff
@@ -15,49 +15,48 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            # wmllint: local spelling unthreatened
-            story= _ "It was a time of doom and great deeds, of fire and blood and slaughter. The men and dwarves of the city fought valiantly. But there were always, it seemed, more orcs to replace the slain, while the defenders could find no succor. Pleas for help went unheeded in other lands, for they were far away and felt unthreatened by the orcs, and were wrapped up in their own affairs and struggles."
+            story= _ "It was a time of doom and great deeds, of fire and blood and slaughter. The men and dwarves of fought valiantly. But there were always, it seemed, more orcs to replace the slain, while the defenders could find no succor. Pleas for help went unheeded in other lands, men, elves and even dwarves, were wrapped up in their own affairs and struggles or deemed the orcs too far away to bring unto them harm."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:38]:
```suggestion
            story= _ "It was a time of doom and great deeds, of fire and blood and slaughter. The men and dwarves fought valiantly. But there were always, it seemed, more orcs to replace the slain, while the defenders could find no succor. Pleas for help went unheeded in other lands for men, elves and even dwarves, were wrapped up in their own affairs and struggles or deemed the orcs too far away to bring unto them harm."
```

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 9]
```diff
@@ -15,49 +15,48 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "After a year and a day of grim battle the city fell. Its streets ran with gore. The dwarves were pushed back into the caverns of Knalga, while the humans of Dwarven Doors who survived the sack were enslaved by the orcs. Cut off from contact with the surface, the dwarves of Knalga fought on, but knew their utter subjugation could not be averted forever."
+            story= _ "After a year and a day of grim battle, the city fell. The streets of Dwarven Doors ran with gore. The dwarves managed to retreat into the caverns Knalga, but the humans who survived the sack were enslaved. Cut off from contact with the surface, the dwarves of Knalga fought on, but knew their subjugation could not be averted forever."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:43]:
```suggestion
            story= _ "After a year and a day of grim battle, the city fell. The streets of the Dwarven Doors ran with gore. The dwarves retreated into the caverns Knalga, but the humans who survived were enslaved. Cut off from contact with the surface, the dwarves of Knalga fought on, but knew their subjugation could not be averted forever."
```

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 10]
```diff
@@ -15,49 +15,48 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "Dark years passed; human generations grew up in slavery, and came near to forgetting that their ancestors had been free — and might have completely forgotten were it not for a fateful day in 518YW. The day that changed the lives of the people of Dwarven Doors — forever."
+            story= _ "Dark years passed; human generations grew up and died in slavery, and with every passing year they and came close to forgetting that their ancestors had ever been free. They might have completely forgotten were it not for a fateful day. The day that changed the lives of the people of Dwarven Doors — forever."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:48]:
```suggestion
            story= _ "Dark years passed; human generations grew up and died in slavery, and with every passing year came closer to forgetting that their ancestors had ever been free. They might have completely forgotten were it not for a fateful day. The day that changed the lives of the people of Dwarven Doors — forever."
```

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 11]
```diff
@@ -15,49 +15,48 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
             # wmllint: local spelling dispiritedly
-            story= _ "It was an early spring day like any other; the humans — joyless beneath the orcish whip — were dispiritedly planting the yearly crop. All at once, the war-drums of the orcs began pounding; harsh war-cries sounded as orcs mustered to gather their weapons and man their posts. Startled, the peasants looked around in alarm, and that was when they saw him."
+            story= _ "It was an early spring day, in the year 518YW. Like any other day, the humans — joyless beneath the orcish whip — were dispiritedly planting the yearly crop. All at once, the war-drums of the orcs began pounding; harsh war-cries sounded as orcs mustered to gather their weapons and man their posts. Startled, the peasants looked around in alarm, and that was when they saw him."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:54]: The double "day" here reads a bit not that nicely.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 12]
```diff
@@ -73,7 +72,7 @@
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "After that day, nothing could be the same. Hope rose in their hearts like a flame long-smothered but rekindled. The brightest and boldest of them began to plan and to train in secret. And the orcish masters, grown stupid from their own arrogance, did not see."
+            story= _ "After that day, nothing could be the same. Hope rose in their hearts like a flame. A flame that had been long-smothered but was now rekindled. The brightest and boldest of them began to plan and to train in secret. And the orcish masters, grown stupid from their own arrogance, did not see."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:75]: Let's keep the original.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 13]
```diff
@@ -283,7 +282,7 @@
         [message]
             speaker=Garrugch
             # wmllint: local spelling slavemaster
-            message= _ "Idle threats. Time as a slavemaster has made you weak. You have the strength of a goblin, and the brains to match. This land now belongs to my master. Surrender now and he might let you be (<i>his</i>) slave."
+            message= _ "Idle threats. Time as a slavemaster has made you weak. You have the strength of a goblin, and the brains to match. This land now belongs to my master. Surrender now, and he might let you be (<i>his</i>) slave."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:285]: No need to add the comma.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 14]
```diff
@@ -242,7 +241,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "Well, those things will certainly get their attention, and there’s nothing wrong with pitchforks. We easily outnumber the orcs, not to speak of the fact that they are killing each other."
+            message= _ "Well, those things will certainly get their attention, but there’s nothing wrong with pitchforks. We easily outnumber the orcs and they're even now killing each other."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:244]: Can you make that apostrophe the unicode version instead?

**Review by nemaara — CHANGES_REQUESTED** [review]

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg (hunk 14 above)
Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:244]: https://wiki.wesnoth.org/Typography_Style_Guide#Apostrophes

Comment (cooljeanius) [reply data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:244]: (there’s one in "there’s" earlier in the string that you can copy from, btw)

**Comment by TheShadowOfHassen** [comment]
I think I got all the changes, if not it'd be great if you real quick do it. I understand the old timey language feel, I just didn't want the sentences to be so long that it was hard to read.

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 15]
```diff
@@ -20,44 +20,44 @@
...
+            story= _  "The dwarves of Knalga, cannily reckoning the gains from trade, built strong fortifications around Dwarven Doors. The people who called it home, dwarven and human alike, knew that the wealth of their city might arouse envy from afar — hence they kept their weapons sharp, and reckoned themselves well able to fight off any bandit gang or petty warlord that could arise in the thinly-settled Northlands."
+
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "For many years it was indeed so. Dwarven Doors was a peaceful and prosperous place. Until the orcs came..."
+            story= _ "For many years it was indeed so. Dwarven Doors was a peaceful and prosperous place, that is, until the orcs came..."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:29]:
```suggestion
            story= _ "For many years it was indeed so. Dwarven Doors was a peaceful and prosperous place until the orcs came..."
```

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 16]
```diff
@@ -20,44 +20,44 @@
...
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "The raids were mere probes, at first, and the orcs mere rabble. But they grew more numerous, and threatening, and the raiding parties became war-bands and then companies. Then there arose a great warlord among the Bloody Sword tribe, the chieftain called Khazg Black-Tusk; and he raised an army, and besieged Dwarven Doors."
+            story= _ "At first, the orcs were disorganized, a mere rabble that easily fell beneath their blades. Little did the people of Dwarven Doors know that the attacks were merely probing at their defenses. The raids grew more numerous and threatening. And soon the raiding parties became war-bands and then companies. At last, one morning, the people of the Dwarven Doors arose to see a great warlord of the Bloody Sword tribe, Khazg Black-Tusk, and his army surounding their defenses."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:34]:
```suggestion
            story= _ "At first, the incursions were mere rabble that were easily felled. Complacent as they were, the people of the Dwarven Doors ignored the orcs even as the raids grew more numerous and threatening. In time, the raiding parties became companies and full war-bands. Then at last, there arose a mighty warlord of the Bloody Sword tribe, Khazg Black-Tusk, who surrounded the entirety of their defenses with a great host of warriors."
```

**Review by nemaara — COMMENTED** [review]
last 2

**Comment by TheShadowOfHassen** [comment]
It looks pretty good now. It wasn't terrible before but you know what I mean. I'm glad I wasn't the only one that got the vibe that the people of Dwarven Doors had messed up, but it seems a bit out of the blue just to show up in the sentence when the orcs are out. I changed the wording of "complacent as they were," to something around the lines of "Complacent in their skill..."

Looking over the next campaigns there's a lot less narration that needs done so I might be able to do the others in a bigger chunk (Although it seems like everyone is rewriting now, which is great.) There are some sentences in dialog that I thought unnatural and could be reworded and of course there is the whole dwarfpower thing that started this.

**Review by nemaara — APPROVED** [review]

**Pull request merged**